File: python/linear_search.py
array_to_search_through = []
for number in range(1, 1001):
    array_to_search_through.append(number)

def linear_search(value_to_find, array_to_search_through):
    index= -1
    if value_to_find in array_to_search_through:
        for i in array_to_search_through:
            index += 1
            if i == value_to_find:
                return index
    else: return None

def global_linear_search(value_to_find, array_to_search_through):
    result= []
    index= -1
    if value_to_find in array_to_search_through:
        for i in array_to_search_through:
            index += 1
            if i == value_to_find:
                result.append(index)
        return result
    else: return None

File: js/linearSearch.js
const arrayToSearchThrough = [];
for (let i = 1; i <= 1000; i++) {
    arrayToSearchThrough.push(i);
}

exports.linearSearch = function(valueToFind, arrayToSearchThrough) {
    let answer = undefined
    if (arrayToSearchThrough.indexOf(valueToFind) !== -1) {
        for (let i = 0; i < arrayToSearchThrough.length; i++) {  
            if (arrayToSearchThrough[i]== valueToFind){
                answer = i 
              break
            }} return answer;
    }}
    
exports.linearSearchGlobal= function(valueToFind, arrayToSearchThrough){
let answer = []
if (arrayToSearchThrough.indexOf(valueToFind) !== -1) {
    for (let i = 0; i < arrayToSearchThrough.length; i++) {  
        if (arrayToSearchThrough[i]== valueToFind){
            answer.push(i)
        }} return answer;
    }else {return null}}